snappiershot/serializers/json.py

```diff
@@ -351,3 +405,40 @@ def decode_collection(dct: Dict[str, Any]) -> Collection:
         raise NotImplementedError(
             f"Deserialization for the following collection type not implemented: {dct}"
         )
+
+    @staticmethod
+    def decode_pandas(dct: Dict[str, Any]) -> Collection:
+        """ Decode an encoded pandas type.
+
+        This encoded numeric type object must be of the form:
+            {
+              PANDAS_KEY: <type-as-a-string>,
+              PANDAS_VALUE_KEY: [<values>]
+            }
+        The values for the PANDAS_KEY and PANDAS_VALUE_KEY constants are attributes
+          to the `snappiershot.serializers.constants.CustomEncodedPandasTypes` class.
+
+        Args:
+            dct: dictionary to decode
+
+        Returns:
+            Decoded pandas object.
+
+        Raises:
+            NotImplementedError - If decoding is not implement for the given numeric type.
+        """
+        # A special effort is made to avoid importing pandas unless it's really necessary.
+        import pandas as pd
```

Comment:
Same as with the `encode_pandas` function, I think we should try to not explicitly import pandas